devel/simple-web-server/Portfile

```diff
@@ -1,31 +1,31 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
 
 PortSystem          1.0
 PortGroup cmake     1.1
-PortGroup boost     1.0
 
-name	  	    simple-web-server
-version		    3.0.2
-revision            1
+name                simple-web-server
+version             3.1.1
 categories          devel
 license             MIT
 maintainers         { gmail.com:rene.vanpaassen @repagh } openmaintainer
-platforms	    darwin
+platforms           darwin
 description         HTTP and HTTPS server and client library in C++11
 long_description    A very simple, fast, multithreaded, platform independent \
-		    HTTP and HTTPS server and client library implemented \
-		    using C++11 and Boost.Asio. Created to be an easy way \
-		    to make REST resources available from C++ applications.
+                    HTTP and HTTPS server and client library implemented \
+                    using C++11 and Boost.Asio. Created to be an easy way \
+                    to make REST resources available from C++ applications.
 
 homepage            https://gitlab.com/eidheim/Simple-Web-Server
-master_sites	    https://gitlab.com/eidheim/Simple-Web-Server/-/archive/v${version}/
-distname	    Simple-Web-Server-v${version}
+master_sites        https://gitlab.com/eidheim/Simple-Web-Server/-/archive/v${version}/
+distname            Simple-Web-Server-v${version}
+configure.args-append \
+                    -DBUILD_TESTING=OFF
 
-checksums           rmd160 1459afa8f4aae1091e2bf7f45a6e357b95406c79 \
-                    sha256 9997079979c542e49809c4ce20942f2eed60b34505b9e757d08966488d18d319 \
-		    size    70637
+checksums           rmd160 d98709ab324a0eb4e3eeb1d7e067818e3c90a964 \
+                    sha256 f8f656d941647199e0a2db3cb07788b0e8c30d0f019d28e6ee9281bc48db132d \
+                    size    73264
 
-boost.depends_type  build
-boost.version       1.69
+depends_build-append \
+                    port:boost
 
-test.run	    yes
+test.run            yes
```

Comment:
setting up a `test` phase is fine, but you switch off building the tests above in the `configure.args` so this currently doesn't work. 

To me it seems that building the tests/examples actually requires `boost`, but also links to the `openssl` port if present (so that should likely be added as a dependency), and wants a c++11 compatible compiler. The latter can be requested by `compiler.cxx_standard 2011`.

However, these files are not actually installed so I am not sure what the purpose of this exactly is.